CDTDatastoreTests/CDTSessionCookieInterceptorTests.m

```diff
@@ -324,4 +365,91 @@ - (void)testCookieInterceptorRenewsEarly
     XCTAssert([helper currentResponse] == 5);
 }
 
+- (void)basic403Test:(nonnull NSString *)error withReason:(NSString *)reason expectedRequests:(int)expectedRequests
```

Comment:
Will there be other tests which use this method? As it stands it seems a bit pointless with only one call site.

Reply:
@tomblench I've added a test in e01dc2f for verifying 403 status code that is not `credentials_expired` error.

Reply:
@tomblench @ricellis Updated both 403 tests and `basic403Test` method in ebe1749.
.